src/main/java/de/retest/recheck/ui/diff/Alignment.java

```diff
@@ -23,7 +25,8 @@
 	private final Map<Element, Element> alignment;
 	private final Map<Element, Element> expectedMapOfElementTree = new HashMap<>();
 	private final Map<Element, Element> actualMapOfElementTree = new HashMap<>();
-	private final Double minimumMatch = getRequiredMinimumMatch();
+	private final Double minimumMatch =
+			getConfiguredDouble( ELEMENT_MATCH_THRESHOLD_PROPERTY, ELEMENT_MATCH_THRESHOLD_DEFAULT );
```

Comment:
I would move this method into Properties as `getElementMatchThreshhold()` and `getWindowMatchThreshold()`.

Reply:
Ok.